lntest/itest/lnd_test.go

```diff
@@ -30,6 +30,8 @@ import (
 	"github.com/btcsuite/btcutil"
 	"github.com/davecgh/go-spew/spew"
 	"github.com/go-errors/errors"
+	"github.com/golang/protobuf/jsonpb"
```

Comment:
Wrong import path being used if we're going with the direction of forking the proto package.

Reply:
jsonpb change is now in a separate pr, this was only used to print debug statements